File: java/SampleWebDriverTest/src/SampleGoogleSearchTest.java
import org.openqa.selenium.By;
import org.openqa.selenium.Keys;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.testng.Assert;
import org.testng.annotations.*;

public class SampleGoogleSearchTest {

    WebDriver driver;

    @BeforeMethod
    public void setUp() throws Exception {
        System.setProperty("webdriver.chrome.driver", "C://chromedriver.exe");
        driver = new ChromeDriver();
    }

    @AfterMethod
    public void tearDown() throws Exception {
        driver.quit();
    }

    @Test
    public void searchEpamTest() throws Exception {
        driver.get("https://www.google.com");
        driver.findElement(By.id("lst-ib")).sendKeys("EPAM");
        driver.findElement(By.id("lst-ib")).sendKeys(Keys.ESCAPE);
        driver.findElement(By.name("btnK")).click();
        driver.findElements(By.cssSelector("#ires .g a")).get(0).click();
        Assert.assertTrue(driver.getCurrentUrl().startsWith("https://careers.epam."));
    }
}

File: .net/SampleWebDriverTest/SampleWebDriverTest/SampleGoogleSearchTest.cs
using NUnit.Framework;
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;
using System.Linq;

namespace SampleWebDriverTest
{
    [TestFixture]
    public class SampleGoogleSearchTest
    {
        IWebDriver _driver;

        [OneTimeSetUp]
        public void OneTimeSetUp()
        {
            _driver = new ChromeDriver();
        }

        [OneTimeTearDown]
        public void OneTimeTearDown()
        {
            _driver.Quit();
        }

        [Test]
        public void SearchEpamTest()
        {
            _driver.Navigate().GoToUrl("https://www.google.com");
            _driver.FindElement(By.Id("lst-ib")).SendKeys("EPAM");
            _driver.FindElement(By.Id("lst-ib")).SendKeys(Keys.Escape);
            _driver.FindElement(By.Name("btnK")).Click();
            _driver.FindElements(By.CssSelector("#ires .g a")).First().Click();
            Assert.IsTrue(_driver.Url.StartsWith("https://careers.epam."));
        }
    }
}
